packages/shared/src/components/header/FeedExploreHeader.tsx

```diff
@@ -117,7 +122,7 @@ export function FeedExploreHeader({
         )}
         {showDropdown && (
           <span className="ml-auto">
-            {withDateRange.includes(path as OtherFeedPage) && (
+            {shouldShowDropdown && (
```

Comment:
would it make sense to use ExploreHeader here as well?

Reply:
Won't work here since the other is used for mobile display and the layout is different.

Reply:
Would prefer then a more specific naming, like MobileExploreHeading for the new one if it makes sense.

Reply:
Yeah, makes sense. Let's update it.

Reply:
Updated the component now.